src/test/metric_config_test.cpp

```diff
@@ -121,9 +120,44 @@ TEST_F(MetricsConfigTest, ChangedValues) {
     ASSERT_EQ(metricConfig.metricsEnabled, true);
 
     // ASSERT_EQ(metricConfig.endpointsPath, "/newmetrics");
-    ASSERT_EQ(metricConfig.requestSuccessGrpcPredict, true);
-    ASSERT_EQ(metricConfig.requestFailRestModelReady, true);
-    ASSERT_EQ(metricConfig.requestSuccessGrpcModelReady, false);
+    ASSERT_TRUE(metricConfig.enabledFamiliesList.find("ovms_requests_success") != metricConfig.enabledFamiliesList.end());
```

Comment:
`enabledFamiliesList` could be private field

Reply:
Use `isFamilyEnabled`

Reply:
Done.

Reply:
All MetricConfig fields and methods are still public